### PR TITLE
challenged access role integration changes

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-5844

### Change description ###

Challenged Access 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: src/shared/components/case-editor/case-access-utils/index.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export class CaseAccessUtils {
                 roleName = `${accessType}-access-${CaseAccessUtils.ADMIN_ROLE_NAME}`;
                 break;
             default:
-                roleName = `${accessType}-access-${CaseAccessUtils.LEGAL_OPERATIONS_ROLE_NAME}`;
+                roleName = `${accessType}-access-legal-ops`;
```
Comment (connorpgpmcelroy) [inline src/shared/components/case-editor/case-access-utils/index.ts:66]: Can't we change the variable name here that was being used?

File: src/shared/components/case-editor/case-access-utils/index.ts [hunk 2]
```diff
@@ -85,15 +85,15 @@ export class CaseAccessUtils {
         const payload: RoleRequestPayload = {
             roleRequest: {
                 assignerId: assignerId,
-                process: 'specific-access',
-                reference: assignerId,
-                replaceExisting: false
+                process: 'challenged-access',
```
Comment (connorpgpmcelroy) [inline src/shared/components/case-editor/case-access-utils/index.ts:88]: Does this mean specific access requests will no longer work?

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Variable should be used, also looks like this may replace some specific access functionality

File: src/shared/components/case-editor/case-access-utils/index.ts (hunk 2 above)
Comment (udaydenduluri33) [reply src/shared/components/case-editor/case-access-utils/index.ts:88]: Very good point. I did not realise that same function is being used by Specific access. I will change it now

File: src/shared/components/case-editor/case-access-utils/index.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/shared/components/case-editor/case-access-utils/index.ts]: Done

File: src/shared/components/case-editor/case-access-utils/index.ts (hunk 2 above)
Comment (udaydenduluri33) [reply src/shared/components/case-editor/case-access-utils/index.ts]: Done

**Review by connorpgpmcelroy — DISMISSED** [review]
Looks good to me

**Review by tarikdogan-hmcts — DISMISSED** [review]

**Review by tarikdogan-hmcts — APPROVED** [review]

**Pull request merged**